dss/util/state_machine/checkout_states.py

```diff
@@ -0,0 +1,92 @@
+state_machine_def = {
```

Comment:
do we need standard SFN things like `Retry`?

Reply:
Yes, coming soon in a separate PR